### ISSUE TITLE
Validate icon names

### ISSUE THREAD

**Issue by insin** [andreypopp/react-fa#37]
Would you be interested in a PR which implements insin/react-octicon#3 for `react-fa` if my attempt at automating this works out?

**Comment by andreypopp** [comment]
Sure.

btw. see https://github.com/gorangajic/react-icons which ships svg icons (fa included) in one icon - one module fashion (we ended up using it instead).

**Comment by andreypopp** [comment]
Does it mean you want to use webpack (through nwb) to build the lib? Interesting! I long wanted to see if webpack can be used to build lib code.

**Comment by insin** [comment]
You can use `output.library`, `output.libraryTarget` and `externals` to configure a build for lib code in Webpack. This is how I do it in nwb: https://github.com/insin/nwb/blob/master/src/commands/build-umd.js

**Comment by andreypopp** [comment]
Yeah, know about that. Was wondering if there's a workflow with webpack which produces separate modules but processed through (babel-loader, css-loader and so on). So that for example consumers don't have to configure whatever exotic things (like CSS modules for example) you've been using while writing a lib.
